### ISSUE TITLE
Solution for All error in JSParser

### ISSUE THREAD

**Issue by usama0x01** [nahamsec/JSParser#37]
Not sure how it happened, but I had "tornado 6.0.1" on my system.

I uninstalled it
**pip uninstall tornado**
and then installed it again
**pip install tornado==5.1.1**

And now I have "tornado 5.1.1", JSPARSER is running now well!
 use pyhon2 not python3 .

**Comment by baotang2118** [comment]
Yep, you are right, you should use tornado 5.1.1 and pycurl 7.43.0.5. Btw, use the docker is better to avoid conflict with other application.
